dask/dataframe/tests/test_categorical.py

```diff
@@ -270,6 +270,13 @@ def assert_array_index_eq(left, right):
     """left and right are equal, treating index and array as equivalent"""
     assert_eq(left, pd.Index(right) if isinstance(right, np.ndarray) else right)
 
+def test_return_type_known_categories():
```

Comment:
This has some linting errors 

```
dask/dataframe/tests/test_categorical.py:273:1: E302 expected 2 blank lines, found 1
dask/dataframe/tests/test_categorical.py:279:1: W293 blank line contains whitespace
```

which you can see [here](https://travis-ci.org/dask/dask/jobs/423424201#L1658-L1660). You can run this locally with `flake8 dask/dataframe/tests/test_categorical.py`